static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -84,34 +94,112 @@ $hosted-stripe: 28px;
 
 .hosted__video {
     vertical-align: top;
+
+    .vjs-big-play-button .vjs-control-text {
+        display: none;
+    }
+
+    //those control buttons will fade in on video play
+    .vjs-fullscreen-control,
+    .vjs-volume-menu-button,
+    .vjs-progress-control {
+        opacity: 0;
+    }
+
+    &.vjs-paused {
+        .vjs-fullscreen-control,
+        .vjs-volume-menu-button,
+        .vjs-progress-control {
+            @include hosted-fade-out;
```

Comment:
Perhaps I am mistaken - but once paused, would this make it impossible to play again?